app/js/lib/cleave.min.js

```diff
@@ -0,0 +1 @@
+../../../node_modules/cleave.js/dist/cleave.min.js
```

Comment:
We definitely should. The symlink is weak and adds work for linting, as tools believe a minified element is in the source. I'll change this.

That would remove the need to switch from the jshint grunt plugin to the native version, but as discussed IRL, we prefer to keep the switch in order to get the latest versions and be more standard.